Compilation error: Cannot find symbol `TargetStage.NOTIFICATION`

Build fails

```
% gradle build                   

> Task :app:compileJava FAILED
[...]app/src/main/java/ciserver/Pipeline.java:117: error: cannot find symbol
                        notifyObservers(TargetStage.NOTIFICATION, PipelineStatus.InProgress);
                                                   ^
  symbol:   variable NOTIFICATION
  location: class TargetStage
[...]app/src/main/java/ciserver/Pipeline.java:122: error: cannot find symbol
                        notifyObservers(TargetStage.NOTIFICATION, status);
                                                   ^
  symbol:   variable NOTIFICATION
  location: class TargetStage
[...]app/src/main/java/ciserver/Pipeline.java:123: error: cannot find symbol
                        if (status != PipelineStatus.Ok || target == TargetStage.NOTIFICATION) {
                                                                                ^
  symbol:   variable NOTIFICATION
  location: class TargetStage
3 errors

FAILURE: Build failed with an exception.
```

--- a/app/src/main/java/ciserver/Pipeline.java
+++ b/app/src/main/java/ciserver/Pipeline.java
@@ -4,28 +4,28 @@ import java.util.ArrayList;
 import java.util.List;
 
 enum PipelineStatus {
-	Ok,
-	Fail,
-	NotImplemented,
-	NotStarted,
-	InProgress
+    Ok,
+    Fail,
+    NotImplemented,
+    NotStarted,
+    InProgress
 }
 
 enum TargetStage {
-	NONE,
-	PULL,
-	LINT,
-	COMPILE,
-	TESTING,
-	ALL
+    NONE,
+    PULL,
+    LINT,
+    COMPILE,
+    TESTING,
+    ALL
 }
 
 interface PipelineObserver {
-	public void update(TargetStage stage, PipelineStatus status);
+    public void update(TargetStage stage, PipelineStatus status);
 }
 
 interface StageTask {
-	public PipelineStatus execute(String pipelineDir, PushEvent event);
+    public PipelineStatus execute(String pipelineDir, PushEvent event);
 }
 
 /**
@@ -34,127 +34,114 @@ interface StageTask {
  */
 class Pipeline {
 
-	private final String pipelineDir;
-	private final PushEvent event;
-	private List<PipelineObserver> observers = new ArrayList<PipelineObserver>();
-	PipelinePuller puller = new PipelinePuller();
-	PipelineCompiler compiler = new PipelineCompiler("/bin/sh", "gradlew", "build", "-x", "test");
+    private final String pipelineDir;
+    private final PushEvent event;
+    private List<PipelineObserver> observers = new ArrayList<PipelineObserver>();
+    PipelinePuller puller = new PipelinePuller();
+    PipelineCompiler compiler = new PipelineCompiler("/bin/sh", "gradlew", "build", "-x", "test");
 
-	Pipeline(PushEvent event, String pipelineDir) {
-		this.event = event;
-		this.pipelineDir = pipelineDir;
-	}
+    Pipeline(PushEvent event, String pipelineDir) {
+        this.event = event;
+        this.pipelineDir = pipelineDir;
+    }
 
-	/**
-	 * Start the pipeline.
-	 *
-	 * @param target is what stage the pipeline should target.
-	 * @return the status of the executed pipeline. OK if everything went ok.
-	 */
-	public PipelineStatus start(TargetStage target) {
-		// No target
-		if (target == TargetStage.NONE) {
-			return PipelineStatus.Ok;
-		}
+    /**
+     * Start the pipeline.
+     *
+     * @param target is what stage the pipeline should target.
+     * @return the status of the executed pipeline. OK if everything went ok.
+     */
+    public PipelineStatus start(TargetStage target) {
+        // No target
+        if (target == TargetStage.NONE) {
+            return PipelineStatus.Ok;
+        }
 
-		// The reason status is redeclared in each stage is to prevent accidental usage
-		// of a status from a different stage.
+        // The reason status is redeclared in each stage is to prevent accidental usage
+        // of a status from a different stage.
 
-		// Pull
-		{
-			notifyObservers(TargetStage.PULL, PipelineStatus.InProgress);
+        // Pull
+        {
+            notifyObservers(TargetStage.PULL, PipelineStatus.InProgress);
 
-			// Code goes here
-			var status = puller.execute(pipelineDir, event);
+            // Code goes here
+            var status = puller.execute(pipelineDir, event);
 
-			notifyObservers(TargetStage.PULL, status);
-			if (status != PipelineStatus.Ok || target == TargetStage.PULL) {
-				return status;
-			}
-		}
+            notifyObservers(TargetStage.PULL, status);
+            if (status != PipelineStatus.Ok || target == TargetStage.PULL) {
+                return status;
+            }
+        }
 
-		// Lint
-		{
-			notifyObservers(TargetStage.LINT, PipelineStatus.InProgress);
+        // Lint
+        {
+            notifyObservers(TargetStage.LINT, PipelineStatus.InProgress);
 
-			// Code goes here
-			var status = PipelineStatus.Ok;
+            // Code goes here
+            var status = PipelineStatus.Ok;
 
-			notifyObservers(TargetStage.LINT, status);
-			if (status != PipelineStatus.Ok || target == TargetStage.LINT) {
-				return status;
-			}
-		}
+            notifyObservers(TargetStage.LINT, status);
+            if (status != PipelineStatus.Ok || target == TargetStage.LINT) {
+                return status;
+            }
+        }
 
-		// Compile
-		{
-			notifyObservers(TargetStage.COMPILE, PipelineStatus.InProgress);
+        // Compile
+        {
+            notifyObservers(TargetStage.COMPILE, PipelineStatus.InProgress);
 
-			// Code goes here
-			var status = compiler.execute(pipelineDir, event);
+            // Code goes here
+            var status = compiler.execute(pipelineDir, event);
 
-			notifyObservers(TargetStage.COMPILE, status);
-			if (status != PipelineStatus.Ok || target == TargetStage.COMPILE) {
-				return status;
-			}
-		}
+            notifyObservers(TargetStage.COMPILE, status);
+            if (status != PipelineStatus.Ok || target == TargetStage.COMPILE) {
+                return status;
+            }
+        }
 
-		// Test
-		{
-			notifyObservers(TargetStage.TESTING, PipelineStatus.InProgress);
+        // Test
+        {
+            notifyObservers(TargetStage.TESTING, PipelineStatus.InProgress);
 
-			// Code goes here
-			var status = PipelineStatus.NotImplemented;
+            // Code goes here
+            var status = PipelineStatus.NotImplemented;
 
-			notifyObservers(TargetStage.TESTING, status);
-			if (status != PipelineStatus.Ok || target == TargetStage.TESTING) {
-				return status;
-			}
-		}
+            notifyObservers(TargetStage.TESTING, status);
+            if (status != PipelineStatus.Ok || target == TargetStage.TESTING) {
+                return status;
+            }
+        }
 
-		// Notify
-		{
-			notifyObservers(TargetStage.NOTIFICATION, PipelineStatus.InProgress);
+        return PipelineStatus.Ok;
+    }
 
-			// Code goes here
-			var status = PipelineStatus.NotImplemented;
+    /**
+     * Adds an observer to the pipeline.
+     *
+     * @param observer is the observer to notify.
+     */
+    public void addObserver(PipelineObserver observer) {
+        observers.add(observer);
+    }
 
-			notifyObservers(TargetStage.NOTIFICATION, status);
-			if (status != PipelineStatus.Ok || target == TargetStage.NOTIFICATION) {
-				return status;
-			}
-		}
+    /**
+     * Unsubscribes the given observer.
+     *
+     * @param observer the observer to unsubscribe.
+     */
+    public void removeObserver(PipelineObserver observer) {
+        observers.remove(observer);
+    }
 
-		return PipelineStatus.Ok;
-	}
-
-	/**
-	 * Adds an observer to the pipeline.
-	 *
-	 * @param observer is the observer to notify.
-	 */
-	public void addObserver(PipelineObserver observer) {
-		observers.add(observer);
-	}
-
-	/**
-	 * Unsubscribes the given observer.
-	 *
-	 * @param observer the observer to unsubscribe.
-	 */
-	public void removeObserver(PipelineObserver observer) {
-		observers.remove(observer);
-	}
-
-	/**
-	 * Notifies all the observers with a stage and status.
-	 *
-	 * @param stage  is the stage to notify about.
-	 * @param status is the status to notify about.
-	 */
-	public void notifyObservers(TargetStage stage, PipelineStatus status) {
-		for (PipelineObserver observer : observers) {
-			observer.update(stage, status);
-		}
-	}
+    /**
+     * Notifies all the observers with a stage and status.
+     *
+     * @param stage  is the stage to notify about.
+     * @param status is the status to notify about.
+     */
+    public void notifyObservers(TargetStage stage, PipelineStatus status) {
+        for (PipelineObserver observer : observers) {
+            observer.update(stage, status);
+        }
+    }
 }

